OpenPNM/Geometry/models/pore_volume.py

```diff
@@ -7,7 +7,244 @@
+    return centroids
+
+
+def _get_vertex_range(verts):
+    "Find the extent of the vetrices"
+    [vxmin, vxmax, vymin, vymax, vzmin, vzmax] = [1e20, 0, 1e20, 0, 1e20, 0]
+    for vert in verts:
+        if np.min(vert[:, 0]) < vxmin:
+            vxmin = np.min(vert[:, 0])
+        if np.max(vert[:, 0]) > vxmax:
+            vxmax = np.max(vert[:, 0])
+        if np.min(vert[:, 1]) < vymin:
+            vymin = np.min(vert[:, 1])
+        if np.max(vert[:, 1]) > vymax:
+            vymax = np.max(vert[:, 1])
+        if np.min(vert[:, 2]) < vzmin:
+            vzmin = np.min(vert[:, 2])
+        if np.max(vert[:, 2]) > vzmax:
+            vzmax = np.max(vert[:, 2])
+    return [vxmin, vxmax, vymin, vymax, vzmin, vzmax]
+
+
+def _get_fibre_image(network, cpores, vox_len, fibre_rad):
+    r"""
+    Produce image by filling in voxels along throat edges using Bresenham line
+    Then performing distance transform on fibre voxels to erode the pore space
+    """
+
+    cthroats = network.find_neighbor_throats(pores=cpores)
+
+    "Below method copied from geometry model throat.vertices"
+    "Needed now as network may not have all throats assigned to geometry"
+    "i.e network['throat.vertices'] could return garbage"
+    verts = _sp.ndarray(network.num_throats(), dtype=object)
+    for i in range(len(verts)):
+        verts[i] = _sp.asarray(list(network["throat.vert_index"][i].values()))
+    cverts = verts[cthroats]
+    [vxmin, vxmax, vymin, vymax, vzmin, vzmax] = _get_vertex_range(cverts)
+    "Translate vertices so that minimum occurs at the origin"
+    for index in range(len(cverts)):
+        cverts[index] -= np.array([vxmin, vymin, vzmin])
+    "Find new size of image array"
+    cdomain = np.around(np.array([(vxmax-vxmin), (vymax-vymin), (vzmax-vzmin)]), 6)
+    logger.info("Creating fibre domain range: " + str(np.around(cdomain, 5)))
+    lx = np.int(np.around(cdomain[0]/vox_len)+1)
+    ly = np.int(np.around(cdomain[1]/vox_len)+1)
+    lz = np.int(np.around(cdomain[2]/vox_len)+1)
+    "Try to create all the arrays we will need at total domain size"
+    try:
+        pore_space = np.ones([lx, ly, lz], dtype=np.uint8)
+        fibre_space = np.zeros(shape=[lx, ly, lz], dtype=np.uint8)
+        dt = np.zeros([lx, ly, lz], dtype=float)
+        # Only need one chunk
+        cx = cy = cz = 1
+        chunk_len = np.max(np.shape(pore_space))
+    except:
+        logger.info("Domain too large to fit into memory so chunking domain to" +
+                    "process image, this may take some time")
+        # Do chunking
+        chunk_len = 100
+        if (lx > chunk_len):
+            cx = np.ceil(lx/chunk_len).astype(int)
+        else:
+            cx = 1
+        if (ly > chunk_len):
+            cy = np.ceil(ly/chunk_len).astype(int)
+        else:
+            cy = 1
+        if (lz > chunk_len):
+            cz = np.ceil(lz/chunk_len).astype(int)
+        else:
+            cz = 1
+
+    "Get image of the fibres"
```

Comment:
Let's be consistent between using strings and comments for annotating code. Since most of the code base uses comments, let's use those.